test/compliance/test_kaldi.py

```diff
@@ -202,7 +207,29 @@ def get_output_fn(sound, args):
                 window_type=args[21])
             return output
 
-        self._compliance_test_helper('fbank', 97, 22, get_output_fn)
+        self._compliance_test_helper(self.test_filepath, 'fbank', 97, 22, get_output_fn, atol=1e-3, rtol=1e-1)
+
+    def test_resample_waveform(self):
+        def get_output_fn(sound, args):
+            output = kaldi.resample_waveform(sound, args[1], args[2])
+            return output
+
+        self._compliance_test_helper(self.test_8000_filepath, 'resample', 32, 3, get_output_fn, atol=1e-2, rtol=1e-5)
+
+    def test_resample_waveform_upsample(self):
```

Comment:
Test 219, 224, and 229, are only checking that the size of the modified signals are correct. Can you change the names to something like `test_resample_waveform_verify_upsample_size` to make that clear?